tests/performance_tests/performance_test_basic.py

```diff
@@ -21,26 +21,144 @@
 relaunchTimeout = 30
 emptyBlockGoal = 5
 
-def waitForEmptyBlocks(node1):
+class blockData():
+    def __init__(self):
+        self.transactions = 0
+        self.cpu = 0
+        self.net = 0
+        self.elapsed = 0
+        self.time = 0
+        self.latency = 0
+
+class chainData():
+    def __init__(self):
+        self.blockLog = []
+    def sum_transactions(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.transactions
+        return total
+    def sum_cpu(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.cpu
+        return total
+    def sum_net(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.net
+        return total
+    def sum_elapsed(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.elapsed
+        return total
+    def sum_time(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.time
+        return total
+    def sum_latency(self):
+        total = 0
+        for block in self.blockLog:
+            total += block.latency
+        return total
+    def print_stats(self):
+        print("Chain transactions: ", self.sum_transactions())
+        print("Chain cpu: ", self.sum_cpu())
+        print("Chain net: ", self.sum_net())
+        print("Chain elapsed: ", self.sum_elapsed())
+        print("Chain time: ", self.sum_time())
+        print("Chain latency: ", self.sum_latency())
+
+class chainsContainer():
+    def __init__(self):
+        self.preData = chainData()
+        self.totalData = chainData()
+        self.startBlock = 0
+        self.ceaseBlock = 0
+    def total_transactions(self):
+        return self.totalData.sum_transactions() - self.preData.sum_transactions()
+    def total_cpu(self):
+        return self.totalData.sum_cpu() - self.preData.sum_cpu()
+    def total_net(self):
+        return self.totalData.sum_net() - self.preData.sum_net()
+    def total_elapsed(self):
+        return self.totalData.sum_elapsed() - self.preData.sum_elapsed()
+    def total_time(self):
+        return self.totalData.sum_time() - self.preData.sum_time()
+    def total_latency(self):
+        return self.totalData.sum_latency() - self.preData.sum_latency()
+    def print_stats(self):
```

Comment:
Add a `__str__(self)` method here, as above.  Consider replacing `print_stats()` entirely with the special method name.  Then down at the bottom, you can simply call `print(cont)`.